Backend/lib/beckn-spec/src/Beckn/Types/Core/Taxi/Update/UpdateEvent/UpdateEventType.hs

```diff
@@ -19,12 +19,14 @@ import Kernel.Prelude
 
 data UpdateEventType
   = PAYMENT_COMPLETED
+  | EDIT_LOCATION
   deriving (Show, Eq, Ord, Read, Generic, ToSchema)
 
 -- Generic instances for type with single value will not work
 instance FromJSON UpdateEventType where
   parseJSON (String "PAYMENT_COMPLETED") = pure PAYMENT_COMPLETED
-  parseJSON (String _) = parseFail "Expected \"PAYMENT_COMPLETED\""
+  parseJSON (String "EDIT_LOCATION") = pure EDIT_LOCATION
```

Comment:
you can use `deriving (FromJSON, ToJSON)` for `UpdateEventType`
it works wrong only for enumerations with single item like `data SingleValue = SingleValue`